go.mod

```diff
@@ -8,9 +8,9 @@ require (
 	go.uber.org/atomic v1.8.0 // indirect
 	go.uber.org/multierr v1.7.0 // indirect
 	go.uber.org/zap v1.20.0
-	k8s.io/api v0.21.8
-	k8s.io/apimachinery v0.21.8
-	k8s.io/client-go v0.21.5
+	k8s.io/api v0.21.9
+	k8s.io/apimachinery v0.21.9
+	k8s.io/client-go v0.21.9
 	sigs.k8s.io/controller-runtime v0.9.7
```

Comment:
Can we update this?

Reply:
Upgrading controller runtime will also upgrade `k8.io/*` .
Our prod is running on `1.20`,  so I avoided it.
Should I still upgrade it?

Reply:
Okay that sounds good to me.